src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,258 @@
+package Duke;
+
+import Duke.Exception.InvalidTaskException;
+import Duke.Exception.MissingDescriptonException;
+import Duke.Exception.MissingDoneNumberFieldException;
+import Duke.Exception.MissingTimeFieldException;
+import Duke.TaskTypes.Deadline;
+import Duke.TaskTypes.Event;
+import Duke.TaskTypes.Task;
+import Duke.TaskTypes.Todo;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    public static final int MAX_NO_OF_TASKS = 100;
+    public static final String TODO = "todo";
+    public static final String DEADLINE = "deadline";
+    public static final String EVENT = "event";
+    public static final String LIST = "list";
+    public static final String DONE = "done";
+    public static final int INITIAL_NO_OF_TASKS = 0;
+
+    public static void main(String[] args) {
+        displayHello();
+        Task[] taskList = initializeTaskList();
+        Scanner myInput = initializeScanner();
+        String userInput = getUserInput(myInput);
+        int numberOfTasks = INITIAL_NO_OF_TASKS;
+        while (isNotBye(userInput)) {
+            String[] splitUserInput = splitTheUserInput(userInput);
+            numberOfTasks = doTaskAndGetNewNumberOfTasks(taskList, numberOfTasks, userInput, splitUserInput);

```

Comment:
I would suggest using a simpler(short and clear) name for the method *doTaskAndGetNewNumberOfTasks(...)*

Reply:
Alright! Hmm might need to come up with a better name for this